src/Features/VisualBasic/Portable/RemoveUnusedParametersAndValues/VisualBasicRemoveUnusedParametersAndValuesDiagnosticAnalyzer.vb

```diff
@@ -35,6 +35,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.RemoveUnusedParametersAndValues
                                                           End Function)
         End Function
 
+        Protected Overrides Function IsIfConditionalDirective(node As SyntaxNode) As Boolean
+            Return TryCast(node, IfDirectiveTriviaSyntax) IsNot Nothing
```

Comment:
humorously enough, given our previous conversation, this *is* a suitable SyntaxFact :D

Reply:
haha.  Fair enough! I def approve of not making life harder for Sam :D